api/report_server.thrift

```diff
@@ -193,20 +192,17 @@ service codeCheckerDBAccess {
                                    3: optional Encoding encoding)
                                    throws (1: shared.RequestFailed requestError),
 
-  // suppress the bug
-  bool suppressBug(1: list<i64> runIds,
-                   2: i64 reportId,
-                   3: string comment)
-                   throws (1: shared.RequestFailed requestError),
-
-  // unsuppress the bug
-  bool unSuppressBug(1: list<i64> runIds,
-                     2: i64 reportId)
-                     throws (1: shared.RequestFailed requestError),
+  // change review status of a bug.
+  bool changeReviewStatus(1: i64 reportId,
+                          2: i64 status,
```

Comment:
If I put the `ReviewStatus` type to this file, there will be a conflict in the `client_db_access_handler.py` because it imports the orm_model module which indicates a conflict, because the table name is the same as the type.